Application/EdFi.Ods.Api/Middleware/EdFiOAuthAuthenticationHandler.cs

```diff
@@ -33,9 +33,8 @@ public class EdFiOAuthAuthenticationHandler : AuthenticationHandler<Authenticati
             IOptionsMonitor<AuthenticationSchemeOptions> options,
             ILoggerFactory loggerFactory,
             UrlEncoder encoder,
-            ISystemClock clock,
             IOAuthTokenAuthenticator oauthTokenAuthenticator)
-            : base(options, loggerFactory, encoder, clock)
+            : base(options, loggerFactory, encoder)
```

Comment:
Fix obsolete warnings related to [ISystemClock](https://learn.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/8.0/isystemclock-obsolete)